sdcm/nemesis.py

```diff
@@ -1015,7 +1015,7 @@ def disrupt_destroy_data_then_rebuild(self):  # pylint: disable=invalid-name
         self.repair_nodetool_rebuild()
 
     def disrupt_nodetool_drain(self):
-        result = self.target_node.run_nodetool("drain", timeout=3600, coredump_on_timeout=True)
+        result = self.target_node.run_nodetool("drain", timeout=15*60, coredump_on_timeout=True)
```

Comment:
I'd rather not to do that - for me it's not bringing much value.

Reply:
Having duplication in 5 different places has bigger value for you? Really?
It is simple change and next time it will require update of just 1 string.

Reply:
It's `find everywhere` for simple pattern and replacing wherever needed. We don't know if we will want doing that in future for all the cases.

I'm not finding it as real issue.

Reply:
In theory @vponomaryov is right, but here I don't know if it brings much value.
I don't think there is one TIMEOUT that is correct for all nodetool drain commands and it's even less readable when you want to know what's the value of that timeout.
So, for now, I'm ok with this as is.

Reply:
Ok